README.md

```diff
@@ -20,92 +20,130 @@ nvm install stable
 yarn
 ```
 
-### Terminal 1 - Backend
+### Terminal 1 & 2 - Backend
+
+Build the backend binaries by running the following:
 
 ```
 cd backend
 cargo build --release
-./target/release/telemetry --help
 ```
 
-By default, telemetry will listen on the local interface only (127.0.0.1) on port 8000. You may change both those values with the `--listen` flag as shown below:
+And then, in two different terminals, run:
+
+```
+./target/release/telemetry_core
+```
+
+and
 
 ```
-telemetry --listen 0.0.0.0:8888
+./target/release/telemetry_shard
 ```
 
-This example listen on all interfaces and on port :8888
+Use `--help` on either binary to see the available options.
+
+By default, `telemetry_core` will listen on 127.0.0.1:8000, and `telemetry_shard` will listen on 127.0.0.1:8001, and expect the `telemetry_core` to be listening on its default address. To listen on different addresses, use the `--listen` option on either binary, for example `--listen 0.0.0.0:8000`. The `telemetry_shard` also needs to be told where the core is, so if the core is configured with `--listen 127.0.0.1:9090`, remember to pass `--core 127.0.0.1:9090` to the shard, too.
 
-### Terminal 2 - Frontend
+### Terminal 3 - Frontend
 
 ```sh
 cd frontend
 yarn install
 yarn start
 ```
 
-### Terminal 3 - Node
+Once this is running, you'll be able to navigate to [http://localhost:3000](http://localhost:3000) to view the UI.
+
+### Terminal 4 - Node
 
 Follow up installation instructions from the [Polkadot repo](https://github.com/paritytech/polkadot)
 
+If you started the backend binaries with their default arguments, you can connect a node to the shard by running:
+
 ```sh
-polkadot --dev --telemetry-url ws://localhost:8000/submit
+polkadot --dev --telemetry-url 'ws://localhost:8001/submit 1'
```

Comment:
could we please mention why it expects ` 1` at the end of the url and what it means?

Reply:
That is documented in substrate/polkadot, but sure, why not.

Reply:
I added a note about this verbosity level (I know it's a strange format, but in part I think it's because you can provide multiple `--telemetry-url` arguments, each with a different verbosity).